dashboard/src/index.css

```
@@ -133,6 +139,51 @@
   font-display: swap;
 }
 
+@font-face {
+  font-family: 'MaiRetroText';
```

Stylelint (17.13.0)

[error] Expected no quotes around "MaiRetroText" (font-family-name-quotes)

(font-family-name-quotes)